packages/webview_flutter/test/webview_flutter_test.dart

```diff
@@ -290,7 +320,7 @@ class FakePlatformWebView {
   List<String> history = <String>[];
   int currentPosition = -1;
 
-  String get currentUrl => history.isEmpty ? null : history[currentPosition];
+  String get url => history.isEmpty ? null : history[currentPosition];
```

Comment:
Curious why did you prefer to rename this to `url`? (especially as now this is what we return for 'currentUrl' method channel invocations)